sources/_sector/dex/trades/base/_sources.yml

```diff
@@ -104,3 +104,7 @@ sources:
     tables:
       - name: LeetSwapV2Factory_evt_PairCreated
       - name: LeetSwapV2Pair_evt_Swap
+  - name: soswap_base
+    tables:
+      - name: SwapFactory_evt_PairCreated
```

Comment:
This source name is different from the source used in https://github.com/duneanalytics/spellbook/pull/5431/files#diff-1baa9bbf0c4d8da8a691b07ffc4682d8be478d34a51e859844a87d5b7cc581f4R19